### ISSUE TITLE
Reproducing the results

### ISSUE THREAD

**Issue by kyungjin-lee** [kuai-lab/cvpr25_3D-GSW#2]
Thank you for this great work and open-sourcing it. The code runs well following the instruction but I am having some difficulty reproducing the results. I used the run code provided in the scripts folder. 

The bit accuracy of the watermark is high as shown in the paper but the quality metrics are low.
Specifically, for the bicycle scene in mip-360 the average bit accuracy on the test set is 0.98 but the PSNR value is only 25.

What could be the reason? Also, would it be possible to provide the watermarked ply files?

Thank you!

**Comment by uuunnnn** [comment]
I got the same result.

**Comment by jang-1996** [comment]
The quality evaluation is conducted by comparing the rendered images from the pre-trained model. This evaluation protocol follows the approach introduced in CopyRNeRF, the first NeRF-based digital watermarking method.

**Comment by jang-1996** [comment]
Closing this issue due to inactivity. Please feel free to reopen.
